<launch>
  <!-- 开启RGB摄像头  -->
  <include file="$(find usb_cam)/launch/usb_cam-test.launch" />
 
  <include file='$(find table_streeing_arm_moveit_config)/launch/demo.launch' />

  <include file="$(find table_arm)/launch/base_serial.launch" />

  <node name='pick_color' pkg="table_arm" type="color_sort_execute" output="screen"/>
 
  <node name='color_hsv_adjust' pkg="table_arm" type="color_hsv_adjust.py" output="screen"/>

</launch>
